daemon/container_unix_test.go

```diff
@@ -24,6 +24,7 @@ func TestContainerWarningHostAndPublishPorts(t *testing.T) {
 			"8080": []nat.PortBinding{{HostPort: "8989"}},
 		}, warnings: []string{"Published ports are discarded when using host network mode"}},
 	}
+	muteLogs()
```

Comment:
LOL, not sure if this is the best way, but it worked. Better suggestions are welcome, but this was small and simple